lib/src/browser/web_compiler.dart

```diff
@@ -0,0 +1,16 @@
+@JS()
+library web_compiler;
+
+import 'package:js/js.dart';
+
+@JS(r'$dartLoader')
+external dynamic get _dartLoader;
+
+/// The compiler used to compile the Dart to JS, determined at runtime
+/// via a global window property that is only present when DDC is used.
```

Comment:
This reads to me as though this is going to return something representing the compiler. Since it returns a boolean indicating whether it was compiled with DDC or not, maybe a sentence like, "Whether or not the Dart dev compiler (DDC), as opposed to the production compiler (dart2js), was used to compile the app." Then I think we could include the explanation that this is determined via the global window property in the paragraph below.